pkg/assume/assume.go

```diff
@@ -204,7 +205,19 @@ func AssumeCommand(c *cli.Context) error {
 
 	} else {
 		creds, err := profile.AssumeTerminal(c.Context, configOpts)
+		connectedToGrantedFlag := false
 		if err != nil {
+			serr, ok := err.(*smithy.OperationError)
+			if ok && connectedToGrantedFlag {
+				if serr.ServiceID == "SSO" {
+					baseUrl, ruleId := "granted.dev", "rul_2BtW97o6jTacUuzxNJZorACn5v0"
```

Comment:
This hardcoded rule needs to be removed

Reply:
It looks like none of this is in use anyway as `connectedToGrantedFlag` is hardcoded to `false`? Can we remove all of this?